From 03e4fdf83717872e349215c64ff8a1901a5bec2d Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Fri, 6 Nov 2020 13:28:56 -0800
Subject: [PATCH 14/57] SystemUI: Remove background tint from emergency tile

Translucent colors don't work well with tinting and our background color
is the same for this tile anyway.

Change-Id: I08b46bb8127551ad438f2643a383c7d88ded36bd
---
 .../com/android/systemui/globalactions/GlobalActionsDialog.java | 2 --
 1 file changed, 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/globalactions/GlobalActionsDialog.java b/packages/SystemUI/src/com/android/systemui/globalactions/GlobalActionsDialog.java
index 70256931964..0851858ed31 100644
--- a/packages/SystemUI/src/com/android/systemui/globalactions/GlobalActionsDialog.java
+++ b/packages/SystemUI/src/com/android/systemui/globalactions/GlobalActionsDialog.java
@@ -939,8 +939,6 @@ public class GlobalActionsDialog implements DialogInterface.OnDismissListener,
                 Context context, View convertView, ViewGroup parent, LayoutInflater inflater) {
             View v = super.create(context, convertView, parent, inflater);
             int textColor;
-            v.setBackgroundTintList(ColorStateList.valueOf(v.getResources().getColor(
-                    com.android.systemui.R.color.global_actions_emergency_background)));
             textColor = v.getResources().getColor(
                     com.android.systemui.R.color.global_actions_emergency_text);
             TextView messageView = v.findViewById(R.id.message);
-- 
2.31.1

